apps/payments/next/middleware.ts

```diff
@@ -20,6 +20,9 @@ export function middleware(request: NextRequest) {
   const PAYPAL_SCRIPT_URL = 'https://www.paypal.com';
   const PAYPAL_API_URL = process.env.CSP__PAYPAL_API;
   const PAYPAL_OBJECTS = 'https://www.paypalobjects.com';
+  const LOCAL_DEV_AVATAR = process.env.NODE_ENV
+    ? 'http://localhost:1111/v1/avatar/d'
+    : '';
```

Comment:
What should the value be for Stage/Production?

Also for local dev it seems the URL changes depending on username and once the user uploads an image.

Reply:
The URL is different on each: 
- local: http://localhost:1111/v1
- stage: https://profile.stage.mozaws.net/v1
- prod: https://profile.accounts.firefox.com/v1

I don't see an existing way that we're indicating which deployment env we're in, that's accessible in the middleware. Is there a good way to check this that I'm missing?

Reply:
I am aware of `NODE_ENV` being one of `'development' | 'production' | 'test'`, though that skips staging AFAICT. Is it fine to omit the staging env url and just set it to be either `http://localhost:1111` or `https://profile.accounts.firefox.com`, depending on the `NODE_ENV` val?

Reply:
`env` vars is the way to go. `middleware.ts` already makes use of a couple, e.g. `CSP__ACCOUNTS_STATIC_CDN` and `CSP__PAYPAL_API`.

`NODE_ENV` won't work for this, since Stage and Production both run with `NODE_ENV === 'production'` and the CSP urls are different for Stage and Production.

Reply:
Got it, makes sense! I'll pull from/add to the env vars where for this

Reply:
we've got the `PROFILE_CLIENT_URL` already defined, so I'm making use of that